frontend/src/ts/const.ts

```diff
@@ -411,9 +411,46 @@ export enum MetadataPlatform {
 
 export enum MetadataValueType {
   STRING = 'String',
-  NUMBER = 'Number',
-  BOOLEAN = 'Boolean',
-  DATETIME = 'DateTime',
+  INTEGER = 'Integer',
+  DOUBLE = 'Double',
+  FLOAT = 'Float',
+}
+
+export enum MetadataParameterType {
```

Comment:
should we share those models in frontend and backend projects?

Reply:
link file can't loader by frontend with **enum** types.